### ISSUE TITLE
Feature suggest - Accessing existing files - matcher discussion

### ISSUE THREAD

**Issue by jy95** [dularion/streama#593]
Hello,

I wrote a scanning library that might be useful for : [MediaScan](https://github.com/jy95/mediaScan).
Reading the [Accessing existing files](https://github.com/dularion/streama#accessing-existing-files) , I saw that its huge advantages aren't here : 
- Extraction of properties you want (quality , language , etc)
- Filtering operations 
- ...

Don't hesistate to ask any question

**Comment by Jeronimo95** [comment]
Are you prefixing all your commit messages with emoji?? That is... fucking great lmao.

**Comment by jy95** [comment]
Yeah : I extended the eslint commit convention with emoji and everything is working great with semantic-release that does all the paintful job of release ^^
